cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -56,7 +61,30 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi) {
     theColumns.push('time');
   }
 
-  var collapseCount = 0;
+  angular.forEach($scope.displayOptions, (value, key) => {
+    this.configOptions[key] = value;
+  });
+
+  this.logEvents = ['ERROR', 'WARN', 'INFO', 'DEBUG', 'TRACE'];
+
+  let included = {
+    'ERROR' : false,
+    'WARN' : false,
+    'INFO' : false,
+    'DEBUG' : false,
+    'TRACE' : false
```

Comment:
Isn't that wrong? The default is you are showing all log levels. That means everything should be selected